templates/dashboard.html

```diff
@@ -98,7 +98,7 @@ <h3>{{gettext("Modeled vs. actual token release schedule")}}</h3>
         <div class="col-md-6">
           <h3>{{gettext("Locked Tokens")}}</h3>
           <div class="desc">
-            {{gettext("Fusce condimentum ultrices est quis dapibus. Phasellus massa tortor, ultricies a imperdiet in, dapibus ac ligula. Duis suscipit odio sed sagittis finibus. Curabitur justo neque, scelerisque eu velit interdum, finibus varius purus.")}}
+            {{gettext("Many of our investors and team members have voluntarily increased their lock-ups. Locked tokens include tokens that otherwise would have been released and entered the circulating supply. This number also includes OGN that is locked up by users in Origin Deals.")}}
```

Comment:
Should line 116 copy be updated to also mention Origin Deals users? Right now it only says "Investors & team members with locked tokens"

Reply:
+1 to including staked Origin Deals tokens in the count. We'll need to add an endpoint to share that data w/ the dashboard.

Reply:
Done.